vpc-endpoint-service.tf

```diff
@@ -8,6 +8,7 @@ resource "aws_vpc_endpoint_service" "hms_readonly" {
   network_load_balancer_arns = ["${aws_lb.apiary_hms_ro_lb.arn}"]
   acceptance_required        = false
   allowed_principals         = formatlist("arn:aws:iam::%s:root", var.apiary_customer_accounts)
+  tags                       = "${merge(map("Name", "${local.instance_alias}-hms-readonly"), "${var.apiary_tags}")}"
```

Comment:
Why do we need to hardcode `Name` tag here when we have the variable `apiary_tags` which can contain `Name` as one of the tags passed from the caller of this module ?

Reply:
As this module is in open-source, shouldn't we leave the choice of tags including the `Name` tag to the user ?

Reply:
`var.apiary_tags` are applied to multiple resources created by this module, and `Name` tag is unique to a resource, so we cannot pass a generic `Name` tag using the variable. 

This is not hardcoding, just like any other variable, we are using `local.instance_alias` to apply appropriate tag.

Reply:
Alright. Thanks for the explanation !